safekeeper/src/http/routes.rs

```diff
@@ -598,6 +600,7 @@ pub fn make_router(conf: SafeKeeperConf) -> RouterBuilder<hyper::Body, ApiError>
         .data(Arc::new(conf))
         .data(auth)
         .get("/metrics", |r| request_span(r, prometheus_metrics_handler))
+        .get("/pprof/profile", |r| request_span(r, pprof_profile_handler))
```

Comment:
I didn't put this behind auth, since it's pretty inconvenient. The perf impact is minor, and only one profile can run at a time, so it's not a DDoS vector. It only exposes function and module names, so it's not a data leak risk.